cluster/manifests/kube-proxy/daemonset.yaml

```diff
@@ -33,9 +33,17 @@ spec:
         image: nonexistent.zalan.do/teapot/kube-proxy:fixed
         args:
         - --hostname-override=$(HOSTNAME_OVERRIDE)
+{{- if ne .Cluster.ConfigItems.experimental_proxy_use_serviceaccount "true" }}
         - --config=/config/kube-proxy.yaml
```

Comment:
Don't we still need the configuration file if running in `serviceaccount` mode?

Reply:
Good point, I'm not sure where this came from!

Reply:
Moved to the correct place (the config).

Reply:
This flag is still guarded by an if :)

Reply:
:(

Reply:
Now it's fixed!